lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -677,7 +677,8 @@ class ModuleInterfaceLoaderImpl {
   std::pair<std::string, std::string> getCompiledModuleCandidates() {
     std::pair<std::string, std::string> result;
     // Should we attempt to load a swiftmodule adjacent to the swiftinterface?
-    bool shouldLoadAdjacentModule = true;
+    bool shouldLoadAdjacentModule = !ctx.IgnoreAdjacentModules ||
+      moduleName == STDLIB_NAME;
```

Comment:
Could we have a comment here about why we except the stdlib?

Reply:
This wasn't needed anymore actually so I removed that commit.